src/app/zap-templates/zcl/data-model/chip/user-label-cluster.xml

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 <configurator>
   <domain name="CHIP"/>
 
-  <struct name="LabelStruct">
+  <struct name="LabelStruct2">
```

Comment:
Please don't do this.  We're going to solve this the right way; this is just adding noise...

Reply:
Boris, send me an email. jonsmirl@gmail.com